third_party/def_parser/BUILD

```diff
@@ -1,5 +1,12 @@
 licenses(["notice"])  # 3-clause BSD
 
+package(
+    default_visibility = [
+        "//src:__subpackages__",
+        "//tools/def_parser:__subpackages__",
```

Comment:
Will this work even though there's an explicit visibility declaration on the :def_parser rule below?

Reply:
Oops, probably not, thanks for spotting this.